RELEASES.md

```diff
@@ -1,3 +1,47 @@
+# Version 1.0.0-alpha.2
+
+## Breaking Changes
+
+### `neon::object::This`
+
+https://github.com/neon-bindings/neon/pull/918
+
+Trait [`neon::object::This`](https://docs.rs/neon/latest/neon/object/trait.This.html) has been removed. `This` was primarily added for use with the `declare_types!` macro to generate classes. The macro was removed and `This` is no longer needed. Additionally, the `This` argument on `JsFunction` was found to be _invalid_ because it asserted at compile time a type for `this` that could change at runtime.
+
+**Note**: This is _not_ unsound because the type would be checked by Node-API and result in a `panic`.
+
+### `JsFunction::this`
+
+https://github.com/neon-bindings/neon/pull/918
+
+`JsFunction::this` was changed to perform a downcast and be _fallible_. This is in line with similar APIs (e.g., `Object::get`). Additionally, an infallible version, `JsValue::this_value` was added that does _not_ perform a downcast.
+
+### Added Feature flag for external buffers
+
+https://github.com/neon-bindings/neon/pull/937
+
+Electron began using [pointer compression](https://www.electronjs.org/blog/v8-memory-cage) on JavaScript values that is incompatible with external buffers. As a preventative measure, `JsArrayBuffer::external` and `JsBuffer::external` have been placed behind a feature flag that warns of Electron incompatibility.
+
+## Improvements
+
+* Lifetimes were relaxed on `execute_scoped` to allow valid code to compile. (https://github.com/neon-bindings/neon/pull/919)
+* Added a `from_slice` helper on `TypedArray` (https://github.com/neon-bindings/neon/pull/925)
+* `JsTypedArray` construction and type aliases (https://github.com/neon-bindings/neon/pull/909)
+
+## Bug Fixes
+
+* Fixed a panic on VM shutdown when using `Channel` (https://github.com/neon-bindings/neon/pull/934)
+* Type tags were added to `JsBox` to reduce the likelihood of undefined behavior when multiple native add-ons are used (https://github.com/neon-bindings/neon/pull/907)
```

Comment:
s/reduce the likelihood of/prevent/

Yes, it's still technically possible but it's vanishingly unlikely. I think it's fair to call this prevention.

Reply:
Neon uses feature detection and type tags are backported all the way to 12. I think this wording is good.